### DISCUSSION TITLE
Universal Python package

### DISCUSSION THREAD

**Discussion by ravishbapna** [opengeos/leafmap#423] [Q&A] [answered]
Just a suggestion to create a universal Python package having functionalities of geemap, leafmap, segment anything, etc.

**Comment by giswqs** [comment, marked as answer]
See

- [geospatial](https://github.com/opengeos/geospatial)
- [geospatial-ml](https://github.com/opengeos/geospatial-ml)
- [pygis](https://github.com/opengeos/pygis)

**Comment by ravishbapna** [comment]
Thank you.
